ios/sound.h

```diff
@@ -48,18 +48,24 @@ class CSound : public CSoundBase
 
     // channel selection
     virtual int     GetNumInputChannels() { return iNumInChanPlusAddChan; }
-    virtual QString GetInputChannelName ( const int iDiD ) { return sChannelNamesInput[iDiD]; }
-    virtual void    SetLeftInputChannel  ( const int iNewChan );
-    virtual void    SetRightInputChannel ( const int iNewChan );
-    virtual int     GetLeftInputChannel()  { return iSelInputLeftChannel; }
-    virtual int     GetRightInputChannel() { return iSelInputRightChannel; }
+    virtual QString GetInputChannelName ( const int iDiD )
+    {
+        return sChannelNamesInput[iDiD];
+    }
```

Comment:
For method definitions with a single statement - no logic - in a header file, I prefer the single line.  They should be the getter/setter-style methods, such as this, mainly.

The rule in this case, however, is that the body remains on a single line if it would exceed the line width, wrapping as a whole to the next line, tabbed in one level.  It looks like that's not captured here.

Reply:
clang-format has
```
AllowAllArgumentsOnNextLine: false
AllowAllConstructorInitializersOnNextLine: false
AllowAllParametersOfDeclarationOnNextLine: false
```
but there's no such setting for 'Blocks'

anyway with `ColumnLimit: 120` that should be gone (at least here)